### DISCUSSION TITLE
[ISSUE] ChatTTS_Speaker 中下载的音色pt文件 怎么使用呢

### DISCUSSION THREAD

**Discussion by LiSongRan** [lenML/Speech-AI-Forge#88] [问答 | Q&A]
### 阅读 README.md 和 dependencies.md

- [X] 我已经阅读过 README.md 和 dependencies.md 文件

### 检索 issue 和 discussion

- [X] 我已经确认之前没有 issue 或 discussion 涉及此 BUG

### 检查 Forge 版本

- [X] 我已经确认问题发生在最新代码或稳定版本中

### 你的issues

使用 ChatTTS_Speaker 中下载的音色pt文件 时遇到了报错

   /ChatTTS-Forge/modules/api/impl/handler/TTSHandler.py", line 31, in __init__     assert isinstance(spk, Speaker), "spk should be Speaker" AssertionError: spk should be Speaker

**Comment by zhzLuke96** [comment]
暂不兼容

如果有 种子 值的话，可以尝试自行创建（webui 和 api均可使用seed重建speaker emb）
